packages/php/email-editor/src/Integrations/Utils/class-styles-helper.php

```diff
@@ -28,10 +28,14 @@ class Styles_Helper {
 	/**
 	 * Parse number value from a string.
 	 *
-	 * @param string $value String value with value and unit.
+	 * @param string|int $value String with value and unit or integer value.
 	 * @return float
 	 */
-	public static function parse_value( string $value ): float {
+	public static function parse_value( string|int $value ): float {
```

Comment:
Please remove this union type because we still need to support PHP 7.4 in WooCommerce

Reply:
Sorry I didn't come back to see the AI results. I could have fixed this sooner.